src/components/pages/DonationForm.vue

```diff
@@ -90,29 +75,32 @@ interface Props {
 	trackingData: TrackingData;
 	campaignValues: CampaignValues;
 	addressValidationPatterns: AddressValidation;
-	startPageIndex: 0 | 1;
+	usesContentCards: true;
```

Comment:
I just noticed this throwing a warning in the dev environment, you should remove it.